### ISSUE TITLE
Angular 10 apps don't build any more

### ISSUE THREAD

**Issue by ch4mpy** [vmware/clarity#4844]
## Describe the bug

I'm working with an Angular workspace (monorepo).
Angular 10 apps built successfully with `@clr/*@next` until yesterday and I can't figure out how to have builds pass again.

## How to reproduce
1. init an angular project
2. import `ClarityModule` in `app.module.ts`
3. build

``` bash
npm update -g @angular/cli
ng new clr-angular10-bug-report --createApplication=false
cd clr-angular10-bug-report/
npm i @clr/angular@next @clr/core@next @clr/icons@next @clr/ui@next
ng g application bogusApp
vim projects/bogusApp/src/app/app.module.ts
```
then replace content with:
``` typescript
import { NgModule } from '@angular/core';
import { BrowserModule } from '@angular/platform-browser';
import { ClarityModule } from '@clr/angular';
import { AppComponent } from './app.component';

@NgModule({
  declarations: [AppComponent],
  imports: [BrowserModule, ClarityModule],
  providers: [],
  bootstrap: [AppComponent],
})
export class AppModule {}
```

then run `ng build`

## Expected behavior

The build should be successful

## Versions

**App**

- Angular: 10.0.5
- Node: v14.0.0 (npm 6.14.7)
- Clarity: 4.0.0-rc.1

**Device:**

- Type: MSI laptop
- OS: Windows 10
- Browser Chrome
- Version 84.0.4147.105

## Additional notes

errors:
```
ERROR in C:/Users/jwacongne/dev/Repos/clr-angular10-bug-report/node_modules/@clr/angular/esm2015/utils/animations/expandable-animation/expandable-animation.js 33:8-43
"export 'ɵɵcomponentHostSyntheticListener' (imported as 'i0') was not found in '@angular/core'

ERROR in C:/Users/jwacongne/dev/Repos/clr-angular10-bug-report/node_modules/@clr/angular/esm2015/utils/animations/expandable-animation/expandable-animation.js 35:8-39
"export 'ɵɵupdateSyntheticHostBinding' (imported as 'i0') was not found in '@angular/core'

ERROR in C:/Users/jwacongne/dev/Repos/clr-angular10-bug-report/node_modules/@clr/angular/esm2015/utils/drag-and-drop/draggable-ghost.js 96:8-39
"export 'ɵɵupdateSyntheticHostBinding' (imported as 'i0') was not found in '@angular/core'
```

**Comment by gnomeontherun** [comment]
We had a bad package deployed in 10.0.0-rc.1 which worked in certain versions of Angular until they refactored something internally, which has been resolved by #4830. The fix is to temporarily downgrade to Angular 10.0.4, or wait for the next release (expected in a few days).

**Comment by ch4mpy** [comment]
I guess you meant 4.0.0-rc.1 (and not 10.0.0-rc.1), but, thanks for work-around.

I also noticed @clr 3.1.5 is compatible with Angular 10, so went back to releases.
